modules/images/webp-uploads/helper.php

```diff
@@ -144,7 +144,13 @@ function webp_uploads_generate_additional_image_source( $attachment_id, $image_s
 
 	// Skip creation of duplicate WebP image if an image file already exists in the directory.
 	if ( file_exists( $destination_file_name ) ) {
-		return new WP_Error( 'webp_image_file_present', __( 'The WebP image already exists.', 'performance-lab' ) );
+		$extension = explode( '|', $allowed_mimes[ $mime ] );
+		$filename  = pathinfo( $destination_file_name, PATHINFO_FILENAME );
+		$filename  = "{$filename}.{$extension[0]}";
+		return array(
+			'file'     => $filename,
```

Comment:
@mukeshpanchal27 is it possible to use something like `wp_basename( $destination_file_name );` here?

It looks like we already run similar logic to create the `$destination_file_name` above?

Reply:
Thank you Joe, Yes it makes sense to use the `wp_basename` function.